provisioning/roles/cuttlefish-app/tasks/main.yml

```diff
@@ -92,3 +92,11 @@
     owner: root
     group: root
     mode: 0744
+
+- name: Create old email archiving cronjob
+  copy:
+    src: archiving
+    dest: /etc/cron.daily/archiving
```

Comment:
That's a good idea. I'm genuinely not sure.

Another thing to check is if there's other scheduled tasks that run at the same time (especially things like backups). I also double-checked that if you have multiple jobs in `cron.daily` they're run sequentially, not concurrently.